geospaas_harvesting/ingesters.py

```diff
@@ -429,6 +428,45 @@ def _get_normalized_attributes(self, url, *args, **kwargs):
         return normalized_attributes
 
 
+class FTPIngester(MetanormIngester):
+    """ Ingester class using FTP to read meta-data from the name of the dataset """
+    LOGGER = logging.getLogger(__name__ + '.FTPIngester')
+
+    def _get_normalized_attributes(self, url, *args, **kwargs):
+        """Gets dataset attributes using ftp"""
+        normalized_attributes = {}
+        raw_attributes={}
+        self.add_url(url, raw_attributes)
+        normalized_attributes = self._metadata_handler.get_parameters(raw_attributes)
```

Comment:
I don't see need to init empty dicts.
```suggestion
        self.add_url(url, {})
        normalized_attributes = self._metadata_handler.get_parameters({})
```

Reply:
This writing style is similar to the writing style of other ingester. For other ingester, the corresponding crawler provides other fields than just only 'url' inside the raw_attributes. Here we only have url inside it. And we have to send this value to `get_parameters`